utilities/getActiveEnv.ts

```diff
@@ -0,0 +1,5 @@
+export const getActiveEnv = () => {
```

Comment:
New utility function to determine what is 'production'. This is a change to the NODE_ENV and we are going to lean on the `CONTEXT` environment var as that is cleaner and clearer.

Reply:
Thanks!